pkg/tsdb/prometheus/promclient/provider.go

```diff
@@ -63,9 +64,11 @@ func (p *Provider) GetClient(headers map[string]string) (apiv1.API, error) {
 		return nil, err
 	}
 
+	httpLogger := httplogger.NewHTTPLogger("prometheus", roundTripper)
```

Comment:
Would be nicer to support this in a more "out of the box" way, but I think we're a bit limited given how Prometheus, CloudWatch, etc. works